x/shared/session.go

```diff
@@ -5,6 +5,10 @@ import (
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
+// closeBlockOffset is the offset to subtract from the earliest supplier commit
+// height to ensure that the supplier has enough time to commit the claim/proof.
+const closeBlockOffset = 1
```

Comment:
This seems like the wrong approach. 

Your goal (in the PR description) is to avoid having the supplier be late.

The on-chain protocol (under x/) sets the rules. 
The off-chain libraries have to follow the protocol.

This laws (protocol) don't bend to servers (web2).

If we have an offset like this, it has to go off-chain.

Reply:
Maybe the PR description is missing nuance.

We are trying to avoid that the protocol imposes an `EarliestSupplierCommitHeight` that is too constraining, making its chances to submit a failing (out of window) transaction non-negiligible.

On a `[10, 20]` commit window it basically prevents the protocol from imposing to the supplier to submit his tx at block `19` or later.